### PR TITLE
Feature/implement cypress testing

### PR DESCRIPTION
**Pull request by lschnell8**
PR Template:

Is this a fix or a feature?
- [ ] Bug fix
- [x] Feature
- [ ] Refactor
 
Describe the feature - fix - refactor:
Implement Cypress testing and Router

How was this tested?
- [ ] open index.html
- [x] localhost
- [ ] terminal

Where should the reviewer start - test:
Test files and components

### CODE REVIEW

File: cypress/integration/1-getting-started/todo.spec.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+//
+// This spec file contains a variety of sample tests
+// for a todo list app that are designed to demonstrate
+// the power of writing tests in Cypress.
+//
+// To learn more about how Cypress works and
+// what makes it such an awesome testing tool,
+// please read our getting started guide:
+// https://on.cypress.io/introduction-to-cypress
+
```
Comment (kavakai) [inline cypress/integration/1-getting-started/todo.spec.js:13]: We should remember to delete the boilerplate tests.

File: src/App.js [hunk 2]
```diff
@@ -15,54 +16,63 @@ class App extends Component{
...
-              movies={this.state.movies}
-              selectMovie={this.selectMovie}
-            />
-          </>
-        ) : (
-            this.state.error && <Error error={this.state.error} navigateHome={this.navigateHome} /> 
-        )}
-      </main>
-    );
+    if (this.state.loading) {
```
Comment (kavakai) [inline src/App.js:43]: Awesome work on figuring this conditional out

File: cypress/integration/1-getting-started/todo.spec.js (hunk 1 above)
Comment (lschnell8) [reply cypress/integration/1-getting-started/todo.spec.js:13]: Good call. We kind of abandoned Cypress for Router. Are we traitors?

**Review by kavakai — APPROVED** [review]
Awesome work on figuring out Router. This was tricky... let's remember that render fires before componentDidMount

**Pull request merged**